traffic_portal/app/src/common/modules/form/serverCapability/form.serverCapability.tpl.html

```diff
@@ -41,6 +41,13 @@
                     <span ng-show="hasError(serverCapabilityForm.name)" class="form-control-feedback"><i class="fa fa-times"></i></span>
                 </div>
             </div>
+            <div class="form-group" ng-class="{'has-error': hasError(serverCapabilityForm.description), 'has-feedback': hasError(serverCapabilityForm.description)}">
+                <label class="control-label col-md-2 col-sm-2 col-xs-12" >Description *</label>
+                <div class="col-md-10 col-sm-10 col-xs-12">
+                    <textarea id="description" name="description" rows="3" cols="17" class="form-control" ng-model="serverCapability.description" maxlength="256" autofocus></textarea>
```

Comment:
This should not have the `autofocus` attribute; the name input already does and appears before it in the form.